ui/ozone/platform/starboard/test/platform_window_starboard_unittest.cc

```diff
@@ -25,36 +25,77 @@ namespace ui {
 namespace {
 class PlatformWindowStarboardTest : public testing::Test {
  public:
-  PlatformWindowStarboardTest() : sb_window_(&delegate_, gfx::Rect(0, 0)) {}
+  PlatformWindowStarboardTest() {}
 
   ~PlatformWindowStarboardTest() = default;
+};
```

Comment:
This doesn't seem to be doing anything, can we remove it and
then s/TEST_F/TEST/ subsequently?